bridge/Mixer.cpp

```diff
@@ -2297,6 +2298,23 @@ bool Mixer::configureBarbellTransport(const std::string& barbellId,
     return true;
 }
 
+bool Mixer::configureBarbellSsrcs(const std::string& barbellId,
+    const std::vector<BarbellStreamGroupDescription>& videoSsrcs,
+    const std::vector<uint32_t>& audioSsrcs)
+{
+    std::lock_guard<std::mutex> locker(_configurationLock);
+    auto barbellItr = _barbells.find(barbellId);
+    if (barbellItr == _barbells.end())
+    {
+        return false;
+    }
+
+    auto& barbell = barbellItr->second;
+    barbell->audioSsrcs = audioSsrcs;
```

Comment:
The barbell will know about all video and audio ssrcs coming in over the barbell.